internal/controller/config/rbac/kube_api_roles.yaml

```diff
@@ -19,8 +19,10 @@ metadata:
 rules:
   - apiGroups:
       - ""
+      - coordination.k8s.io
```

Comment:
Did this come via a marker somewhere that I couldn't find or what?

Reply:
This should have come as part of https://github.com/openshift/assisted-service/commit/ef9e854a1ceb54318e93141ba365e9e0770302e4 which added it here:

https://github.com/openshift/assisted-service/blob/e0a3942e849a1a594f9183700f0ef115e1d48285/config/rbac/leader_election_role.yaml#L9